src/api/settings/getServerSettings.js

```diff
@@ -35,6 +35,10 @@ export type ApiResponseServerSettings = {|
   realm_uri: string,
   require_email_format_usernames: boolean,
   zulip_version: string,
+  // zulip_feature_level added for server v2.2, feature level 1
+  // See https://github.com/zulip/zulip/commit/e3b90a5ec8 for
+  // the server-side implementation.
+  zulip_feature_level?: number,
```

Comment:
This change and the one in `initialDataTypes` are best squashed -- they're two manifestations of the same thing.

A good prefix for the commit-message summary would be `api types:`.